include/wx/uilocale.h

```diff
@@ -40,6 +41,16 @@ class WXDLLIMPEXP_BASE wxLocaleIdent
     // and region separated by dashes.
     static wxLocaleIdent FromTag(const wxString& tag);
 
+    // Add likely subtags to a given locale identifier.
+    static wxLocaleIdent AddLikelySubtags(const wxLocaleIdent& localeIdent);
+
+    // Remove likely subtags from a given locale identifier, favor region.
+    static wxLocaleIdent RemoveLikelySubtags(const wxLocaleIdent& localeIdent, bool favorRegion = true);
+
+    // Find the best match between desired and supported languages/locales.
+    static wxString GetBestMatch(const wxArrayString& desired, const wxArrayString& supported, const wxString& defaultTag);
```

Comment:
Do we use non-empty `defaultTag` anywhere? I can't find it in the diff, but perhaps I'm missing something. But if I don't, can we get rid of this argument?

Reply:
> > Do we use non-empty `defaultTag` anywhere?
> 
> Currently not. But this could change, depending on the answer to my [first question](https://github.com/wxWidgets/wxWidgets/pull/24855#issue-2560038878).

I have to correct myself. At the moment, all tests for method `GetBestMatch()` use the default tag parameter. But this could be changed to check for an empty string instead.